libs/langchain-openai/src/tests/azure/llms.int.test.ts

```diff
@@ -0,0 +1,333 @@
+import { test, expect } from "@jest/globals";
+import { LLMResult } from "@langchain/core/outputs";
+import { StringPromptValue } from "@langchain/core/prompt_values";
+import { CallbackManager } from "@langchain/core/callbacks/manager";
+import { NewTokenIndices } from "@langchain/core/callbacks/base";
+import {
+  ClientSecretCredential,
+  getBearerTokenProvider,
+} from "@azure/identity";
+import { getEnvironmentVariable } from "@langchain/core/utils/env";
+import { AzureOpenAI } from "../../azure/llms.js";
+
+test("Test OpenAI", async () => {
+  const model = new AzureOpenAI({
+    maxTokens: 5,
+    modelName: "gpt-3.5-turbo-instruct",
+  });
+  const res = await model.invoke("Print hello world");
+  console.log({ res });
+});
```

Comment:
Same here about test names and expecting something

Reply:
I have updated the test names. Regarding expecting, I am using the same pattern as other libs. Kindly refer the explanation above. Thanks